src/pow.cpp

```diff
@@ -231,13 +344,115 @@ bool CheckNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader& bl
     return ((min <= target) && (target <= max));
 }
 
-bool CheckProofOfWork(uint256 hash, unsigned int nBits, const Consensus::Params& params)
+bool CheckNextWorkRequiredAux(const CBlockIndex* pindexLast, const CBlockHeader& block, const Consensus::Params &params)
```

Comment:
Nit: I think normally the `&` is before the space, which you're inverting at the last param here.